### PR TITLE
style: Add scale property to Toggle component

### PR DESCRIPTION
**Pull request by heyimbrucelee**
changed checked color from 'success' to 'secondary'

### CODE REVIEW

**Review by RabbitDoge — COMMENTED** [review]
I checked the result. Everything looks good.
Please add a test here https://github.com/pancakeswap/pancake-uikit/blob/master/src/__tests__/components/toggle.test.tsx for the `small` variant , and we can merge

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Looks good. One thing - please use the same sizing pattern we use for other components. Check the `<Checkbox />` component for an example.

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
The color changed has never been requested by our designer, please revert this change

**Comment by RabbitDoge** [comment]
image

For your reference.
Please contact Chef Bugs if you need more details about the design

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
The code is ok, but the pr still doesn't follow our coding rules.
If you don't know / want to use prettier, run `yarn format` before you commit.

File: src/components/Toggle/types.ts [hunk 1]
```diff
@@ -4,4 +4,33 @@ export type ToggleTheme = {
...
+
+export type Scales = typeof scales[keyof typeof scales];
+
+export interface ToggleProps extends InputHTMLAttributes<HTMLInputElement> {
+  scale: Scales;
+}
+
+export interface HandleProps extends InputHTMLAttributes<HTMLInputElement> {
+  scale: Scales;
+}
```
Comment (hachiojidev) [inline src/components/Toggle/types.ts:20]: Why do these extend `HTMLInputElement`?

Comment (heyimbrucelee) [reply src/components/Toggle/types.ts:20]: bad ts knowledge :( , i'll fix it

Comment (heyimbrucelee) [reply src/components/Toggle/types.ts]: i hope this time its ok !

**Review by hachiojidev — APPROVED** [review]

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**